.github/workflows/test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
```

Comment:
Please remove all the wheel files, these should not be in version control.

Reply:
Sure. I will do as soon as there is a new `pyqir` release.

Reply:
I guess we agreed here.